vmdb/app/assets/javascripts/miq_slickgrid.js

```diff
@@ -88,7 +88,7 @@ function TreeFormatter(row, cell, value, columnDef, dataContext) {
 }
 
 function myFilter(item, rows) {
-  if (item.parent != null) {
+  if (item.parent) {
```

Comment:
Possibly wrong, parent may be undefined, null, or a number (including 0), and 0 seems like it should go in.